RELEASE-NOTES.md

```diff
@@ -9,19 +9,17 @@ Packages released:
 - @patternfly/patternfly@v2.71.1
```

Comment:
do we want to update this?

```suggestion
- @patternfly/patternfly@v2.71.3
```

Reply:
Yep, our officially released version should now include the updated release notes since the gatsby-theme-patternfly-org branch has gone away